packages/web_benchmarks/lib/src/runner.dart

```diff
@@ -55,8 +55,15 @@ class BenchmarkServer {
     required this.chromeDebugPort,
     required this.headless,
     required this.treeShakeIcons,
+    this.initialPage = defaultInitialPage,
   });
 
+  /// The default initial page to load upon opening the benchmark app in Chrome.
+  ///
+  /// This value will be used by default if no value is passed to [initialPage].
+  /// See also, [initialPage] and [_benchmarkAppUrl].
```

Comment:
I'm not sure the link to the private `_benchmarkAppUrl` will show up gracefully in the API docs.